### ISSUE TITLE
The ssl3 only natives version requires glibc 2.36

### ISSUE THREAD

**Issue by Shevchik** [tdlight-team/tdlight-java#141]
Ubuntu 22.04 has no openssl1.1 on fresh install, so ssl3 version has to be used. But ubuntu 22.04 uses glibc 2.35, not glibc 2.36, so tdjni.so library fails to load.

I have successfully built tdjni.so on ubuntu 22.04 myself locally, replaced the loader, and verified that it works.
The github actions build seems to use ubuntu 22.04, so I'm not sure from where does the requirement appear.

**Comment by unoexperto** [comment]
@Shevchik Denis, were you able to fix this without manually replacing the binary?

I've built tdlib myself but with that version I get another error

```
[ 1][t 0][1681988104.672035217][tl_jni_object.cpp:39] Can't find class [org/drinkless/tdlib/Client]
FATAL ERROR in native method: Can't find class [org/drinkless/tdlib/Client]
  at jdk.internal.loader.NativeLibraries.load(java.base@19.0.2/Native Method)
```

**Comment by cavallium** [comment]
Fixed in v3.0.3+
